addons/quickmount/functions/fnc_canShowFreeSeats.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author: Dystopian
+ * Checks if Free Seats menu can be shown.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Unit <OBJECT>
+ *
+ * Return Value:
+ * Can show menu <BOOL>
+ *
+ * Example:
+ * [cursorObject, player] call ace_quickmount_fnc_canShowFreeSeats
+ *
+ * Public: No
+ */
+
+params ["_vehicle", "_unit"];
+
+(
+    GVAR(enabled) && {GVAR(enableGetInMenu)}
+    || {!isNull objectParent _unit}
```

Comment:
This means the menu will always show when mounted
I think it should respect the setting
Maybe convert to a 3 way setting, disabled, in-vehicle and always

Reply:
Setting is (`None`, `Get In`, `Change seat`, `All`) now.